packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard-steps/select-country-step/select-country-step.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { ResidenceList } from '@deriv/api-types';
+import { Localize } from '@deriv/translations';
+import CountrySelector from '../../country-selector';
+
+type TCountrySelectorProps = {
+    onSelect: (country: ResidenceList[0]) => void;
```

Comment:
```suggestion
    onSelect: React.ComponentProps<typeof CountrySelector>['onSelect']
```
@hamid-deriv The reason is since we are prop drilling here, If we change the `onSelect` prop signature in `CountrySelector` which is the main component the type will also be changed in parent components, But if we define the type separately in each component then we will need to maintain it in many places as we change something.

Reply:
@farzin-deriv thanks, I got it :+1: